routes/socket/user-events.js

```diff
@@ -2537,6 +2537,12 @@ module.exports.handleUpdatedGameSettings = (socket, passport, data) => {
 			const userIdx = userList.findIndex(user => user.userName === passport.user);
 
 			for (const setting in data) {
+				if (setting == 'blacklist') {
+					while (data[setting].length > 30) {
```

Comment:
This could be done simpler, and with avoiding a unnecessary while by simply replacing it with

```data.blacklist = data.blacklist.slice(0,29);```

Also, I feel like while this is a good solution for adding more blacklists, this doesn't quite resolve the issues brought forth by #1230 fully.

Reply:
I agree that the while is extra, but wouldn't shift() do the same thing as slice(0, 29)?

Reply:
Specifically, I think it resolves this point in #1230: https://github.com/cozuya/secret-hitler/issues/1230#issuecomment-485962822

Reply:
Yeah, it could - unless somehow the blacklist length exceeded 31, but you're right, in most instances the single slice is probably fine. 

And yeah I see, this resolves that one comment but not the entire issue - I just misread that 👍